cubedash/_pages.py

```diff
@@ -244,6 +244,38 @@ def about_page():
     return utils.render("about.html")
 
 
+@app.route("/dashboard")
+def dashboard_page():
+    dc = datacube.Datacube(index=_model.STORE.index)
+    dashboard = []
+    import os
+
+    for p, _ in _model.get_products_with_summaries():
+        dataset_list = dc.find_datasets(
+            product=p.name, limit=100
+        )  # sample max 100 datasets
+        uri_list = []
+        for dl in dataset_list:
+            for uri in dl.uris:
+                uri_list.append(uri)
```

Comment:
This will make page loads quite slow -- we're fetching ~60*100 full datasets from the db.

Since we only care about `uris`, we could fetch just those from the search, rather than the whole datasets:

```suggestion
        # Sample 100 dataset uris
        uri_list = [
            uri
            for [uri] in dc.index.datasets.search_returning(
                ["uri"], product=p.name, limit=100
            )
        ]
```